### DISCUSSION TITLE
Performance Improvement.

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2300] [General]
Hi, everyone, we want to ask how we can greatly speed up the processing times?
OS: Linux | Nvidia RTX: 4000 | GPU: 4GB | System Memory: 8GB
It took 3minutes 45 seconds to generate 10 images; and 8 minutes 51 seconds to generate 25 images for a user. Approximately 22.5 seconds per image.
Which means that our GPU can generate around 115,200 images per month.
Which supports 11,520 user requests at 10 images per user request or 4,608 user requests at 25 images per user request. I'd love to 4x - 5x that for this to be feasible.

Looking forward to hearing back from you. Thanks.

**Comment by mashb1t** [comment]
duplicated by https://github.com/lllyasviel/Fooocus/discussions/2302, @oldhand7 please stop spamming both discussion and issue. Both are actively being monitoried, no worries.
